.changelog/44336.txt

```diff
@@ -0,0 +1,15 @@
+```release-note:new-data-source
+aws_odb_cloud_autonomous_vm_clusters_list
```

Comment:
can we change the names in the CHANGELOG to match the names of the resources by removing `_list` at the end?